From c3e58e2179ea78262879a4b7848139ad73541d68 Mon Sep 17 00:00:00 2001
From: Joe Maples <joe@frap129.org>
Date: Sun, 21 Jan 2018 19:15:40 -0500
Subject: [PATCH 141/296] hwui: Disable fine-grained logging

We're using the default OpenGL renderer, which I don't think anyone plans on debugging
---
 libs/hwui/Android.bp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libs/hwui/Android.bp b/libs/hwui/Android.bp
index c2c9367771f..913ce9c813e 100644
--- a/libs/hwui/Android.bp
+++ b/libs/hwui/Android.bp
@@ -248,7 +248,7 @@ cc_library {
         // Enables fine-grained GLES error checking
         // If enabled, every GLES call is wrapped & error checked
         // Has moderate overhead
-        "hwui_enable_opengl_validation",
+        //"hwui_enable_opengl_validation",
 ],
 }
 
-- 
2.11.0

